requirements.txt

```diff
@@ -0,0 +1,27 @@
+certifi @ file:///home/conda/feedstock_root/build_artifacts/certifi_1720457958366/work/certifi
+chardet @ file:///home/conda/feedstock_root/build_artifacts/chardet_1695468619093/work
+contourpy @ file:///home/conda/feedstock_root/build_artifacts/contourpy_1712429918028/work
+cycler @ file:///home/conda/feedstock_root/build_artifacts/cycler_1696677705766/work
+fonttools @ file:///home/conda/feedstock_root/build_artifacts/fonttools_1720359030700/work
+freetype-py @ file:///home/conda/feedstock_root/build_artifacts/freetype-py_1650983368720/work
+greenlet @ file:///home/conda/feedstock_root/build_artifacts/greenlet_1703201593537/work
+kiwisolver @ file:///home/conda/feedstock_root/build_artifacts/kiwisolver_1695379925569/work
+matplotlib==3.9.1
+munkres==1.1.4
+numpy @ file:///home/conda/feedstock_root/build_artifacts/numpy_1707225359967/work/dist/numpy-1.26.4-cp312-cp312-linux_x86_64.whl#sha256=031b7d6b2e5e604d9e21fc21be713ebf28ce133ec872dce6de006742d5e49bab
+packaging @ file:///home/conda/feedstock_root/build_artifacts/packaging_1718189413536/work
+pandas @ file:///home/conda/feedstock_root/build_artifacts/pandas_1715897630316/work
+pillow @ file:///home/conda/feedstock_root/build_artifacts/pillow_1719903548521/work
+pycairo==1.26.1
+pyparsing @ file:///home/conda/feedstock_root/build_artifacts/pyparsing_1709721012883/work
+python-dateutil @ file:///home/conda/feedstock_root/build_artifacts/python-dateutil_1709299778482/work
+pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1706886791323/work
+rdkit @ file:///home/conda/feedstock_root/build_artifacts/rdkit_1719850005757/work
+reportlab @ file:///home/conda/feedstock_root/build_artifacts/reportlab_1719323846843/work
```

Comment:
I actually have no idea why all of those were generated. Here is what I did: 
I installed miniconda.
Then I ran  
```
conda create -c conda-forge -n my-rdkit-env rdkit=2024.03.4 python=3.12
conda activate my-rdkit-env
``` 

Then I generated requirements file using:
`pip freeze > requirements.txt`

And all those stuff was generated so I leaved it as it was.
I don't know where did so many dependencies come from, any ideas?
Maybe it is because of something I did wrong with conda? I'll do the same thing using venv and lets see.

Reply:
yep, It is because I did sth wrong with conda, bacause using venv, I got only these in requirements.txt:
```
numpy==1.26.4
pillow==10.4.0
rdkit==2024.3.3
```

Reply:
numpy and pillow came with rdkit I guess